kork/kork-artifacts/kork-artifacts.gradle

```diff
@@ -5,14 +5,18 @@ dependencies {
   api(platform(project(":spinnaker-dependencies")))
 
   implementation project(":kork-annotations")
+  implementation project(":kork-aws")
   implementation project(":kork-exceptions")
   implementation project(":kork-security")
   implementation "com.fasterxml.jackson.core:jackson-databind"
+  implementation platform('software.amazon.awssdk:bom')
```

Comment:
A ... concern:  Will this interact with the clouddriver AWS dependencies at all?  I'd assume not since those are set there.. .just wondering if we're breaking any other deps on the platform implementation dep here... And whether this is the "right" spot for that vs. spinnaker-dependencies which should also be applied everywhere?

Reply:
No. This is v2 which clouddriver does not use. Once it does however, we will need to ensure they are compatible